### PR TITLE
Detect "just lapsed" users and register them for a Delighted survey

### PR DESCRIPTION
**Pull request by martinraison**
@stkem @andrewconner 
sbt reload required

### CODE REVIEW

File: kifi-backend/modules/shoebox/app/com/keepit/model/UserValueRepo.scala [hunk 1]
```diff
@@ -16,6 +16,7 @@ trait UserValueRepo extends Repo[UserValue] {
   def getUserValue(userId: Id[User], key: UserValueName)(implicit session: RSession): Option[UserValue]
   def setValue[T](userId: Id[User], name: UserValueName, value: T)(implicit session: RWSession): T
   def clearValue(userId: Id[User], name: UserValueName)(implicit session: RWSession): Boolean
+  def getLastActive(after: DateTime, before: Option[DateTime], maxCount: Int = 100, skipCount: Int = 0)(implicit session: RSession): Seq[Id[User]]
```
Comment (martinraison) [inline kifi-backend/modules/shoebox/app/com/keepit/model/UserValueRepo.scala:19]: This is the first "usecase-specific" function in `UserValueRepo` - couldn't really find a way around that.

**Comment by stkem** [comment]
lgtm

**Comment by stkem** [comment]
@martinraison can this be merged?

**Comment by martinraison** [comment]
yes - I'll rebase and merge

**Pull request merged**